fpga/scripts/program.tcl

```diff
@@ -18,9 +18,14 @@
 open_hw
 
 connect_hw_server -url localhost:3121
-open_hw_target {localhost:3121/xilinx_tcf/Digilent/200300A8CD43B}
+open_hw_target {localhost:3121/xilinx_tcf/Digilent/210203A5FC70A}
```

Comment:
Although that is not part of the standard flow I'd appreciate a switch on board variable as well. So that me and you can use this script :-)